addons/common/functions/fnc_checkVersionNumber.sqf

```diff
@@ -21,20 +21,26 @@ if (canSuspend) exitWith {
 };
 
 params [["_whitelist", missionNamespace getVariable ["ACE_Version_Whitelist", []]]];
-
-private _files = CBA_common_addons select {
-    (_x select [0, 3] != "a3_") &&
-    {_x select [0, 4] != "ace_"} &&
-    {!((toLowerANSI _x) in _whitelist)}
-};
+TRACE_1("",_whitelist);
 
 private _cfgPatches = configFile >> "CfgPatches";
+private _files = [];
 private _versions = [];
 
 {
+    // Skip A3, ACE, and whitelisted addons, because we have already done the check.
+    private _addon = _x;
+    if (
+        (_x select [0, 3] == "a3_") ||
+        {_x select [0, 4] == "ace_"} ||
+        {_x select [0, 12] == "CuratorOnly_"} ||
```

Comment:
What's this change? It's not mentioned in the description of the PR.

Reply:
There are A3 addons starting with `CuratorOnly_`. I thought that if we already skip A3 addons, we can skip these too. Should I revert this?

Reply:
> Should I revert this?

No, it's a good thing to add. I wasn't quite sure what it was as you didn't describe it at first, but it's fine now.